runtime/dart_isolate.h

```diff
@@ -58,7 +59,8 @@ class DartIsolate : public UIDartState {
               fml::RefPtr<flow::SkiaUnrefQueue> unref_queue,
               std::string advisory_script_uri,
               std::string advisory_script_entrypoint,
-              ChildIsolatePreparer child_isolate_preparer);
+              ChildIsolatePreparer child_isolate_preparer,
+              bool terminate_child_isolates = true);
```

Comment:
I wouldn't default these arguments in the constructor. Doing the same in the variable declaration works better.